src/pages/Painel.js

```diff
@@ -41,6 +43,8 @@ export default function ErroLista() {
       {eventos.map((e, index) => {
         return <Eventos key={index} evento={e}></Eventos>;
       })}
+      {/* apenas para teste */}
+      <Link to="/alert/22bPWWkrVE62tLIrLHsH">Pág Alerta</Link>
```

Comment:
Opa, eu tive um probleminha pra testar pq esse id não existe no meu usuário, vamos mudar pra `example` por enquanto, ja que é um id que todo usuário tem por enquanto.

```suggestion
      <Link to="/alert/example">Pág Alerta</Link>
```